Adding tags from model to Inventory

arihantbFirstwave opened the pull request.

arihantbFirstwave committed:
Adding tags from model to Inventory

markdueck1 commented:
This adds a node init into every inventory save which we cannot do. I will rework this.

Sheetalfw committed:
Testing inventory tags with mongo.

Commit:
WIP

Commit:
get tags from model only on update

catchall_inventory needs to be passed as parameter so the SYS inventory and Node use the same data

dataset_tags stored outside of dataset_info in object but in that property in the database.

same for data_tags

Commit:
add tag tests, disable broken inventory tests

Commit:
Merge branch 'nmis9_dev' into feature/omk-11992

Commit:
change parse_model_for_tags params to use model instead of node

Commit:
Merge branch 'feature/omk-11992' of github.com:Opmantek/nmis9 into feature/omk-11992
